code/modules/mob/animations.dm

```diff
@@ -166,6 +171,8 @@ note dizziness decrements automatically in the mob's Life() proc.
 		if(SOUTHWEST)
 			pixel_x_diff = -8
 			pixel_y_diff = -8
+		else
```

Comment:
Did you test what effect this has on attack animations? Because it might stop certain animations from playing.